llama_index/embeddings/__init__.py

```diff
@@ -17,17 +18,18 @@
 from llama_index.embeddings.utils import resolve_embed_model
 
 __all__ = [
-    "GoogleUnivSentEncoderEmbedding",
-    "LangchainEmbedding",
-    "OpenAIEmbedding",
-    "LinearAdapterEmbeddingModel",
     "AdapterEmbeddingModel",
+    "ClarifaiEmbedding",
+    "DEFAULT_HUGGINGFACE_EMBEDDING_MODEL",
+    "ElasticsearchEmbeddings",
+    "GoogleUnivSentEncoderEmbedding",
+    "GradientEmbedding",
     "HuggingFaceEmbedding",
     "InstructorEmbedding",
+    "LangchainEmbedding",
+    "LinearAdapterEmbeddingModel",
+    "OpenAIEmbedding",
     "OptimumEmbedding",
-    "resolve_embed_model",
-    "DEFAULT_HUGGINGFACE_EMBEDDING_MODEL",
     "SimilarityMode",
-    "ElasticsearchEmbeddings",
-    "ClarifaiEmbedding",
+    "resolve_embed_model",
```

Comment:
that's a lot of diff

Reply:
sorting.

Reply:
I wouldn't mind if it was our project, but let's not do that on someone else's code base

Reply:
fair